This patch allows us to install the Vixie-compatible binaries as
non-root without creating /var/run, etc.

--- mcron-1.0.6/makefile.in	2010-06-19 20:44:17.000000000 +0200
+++ mcron-1.0.6/makefile.in	2010-07-04 16:16:25.000000000 +0200
@@ -1004,15 +1004,11 @@ mcron.c : main.scm crontab.scm makefile.
 	@rm -f mcron.escaped.scm > /dev/null 2>&1
 
 install-exec-hook:
-	@if [ "x@NO_VIXIE_CLOBBER@" != "xyes"   -a   "`id -u`" -eq "0" ]; then \
+	@if [ "x@NO_VIXIE_CLOBBER@" != "xyes" ]; then \
        rm -f $(fpp)cron$(EXEEXT) > /dev/null 2>&1; \
        $(INSTALL) --mode='u=rwx' mcron$(EXEEXT) $(fpp)cron$(EXEEXT); \
        rm -f $(fpp)crontab$(EXEEXT) > /dev/null 2>&1; \
        $(INSTALL) --mode='u=rwxs,og=rx' mcron$(EXEEXT) $(fpp)crontab$(EXEEXT); \
-       $(INSTALL) -d --mode='u=rwx' $(DESTDIR)/var/cron; \
-       $(INSTALL) -d --mode='u=rwx,og=rx' $(DESTDIR)/var/run; \
-       $(INSTALL) -d --mode='u=rwx,og=rx' $(DESTDIR)@GUILE_SITE@; \
-       $(INSTALL) -d --mode='u=rwx,og=rx' $(DESTDIR)@GUILE_SITE@/mcron; \
     elif [ "x@NO_VIXIE_CLOBBER@" = "xyes" ]; then \
        echo "Not installing Vixie-style programs"; \
     else \

